docs/HowToUse.md

```diff
@@ -0,0 +1,28 @@
+# How to use?
+
+Let's compile an example by using ONE-vscode.
+
+## Install toolchains
+
+First, you need ONE toolchains. Let's install ONE toolchains. (You should install the in-house backend to follow this.)
+
+<img src="https://user-images.githubusercontent.com/7223627/174947559-71ab213c-68c4-4700-bcec-e55f6a2203bf.gif" width=500 />
+
+## Prepare a NN model and a config file
+
+Second, you need a NN model and a config file to run compiling. Now you can get one from [res/modelDir/truediv](../res/modelDir/truediv/). Add it to your workspace. Now you can see a model and a config.
+
+<img src="https://user-images.githubusercontent.com/10216715/174798969-eee44fea-bd71-4e6a-8e2c-9e1de37ad74a.gif" width=500 />
+
+## Compile models
+
+Last, you can run `compile`.
```

Comment:
The gif is showing a process clicking a button `Run`, but the document says it's `compile`. 
How about 

Last, you can compile the model by pushing `Run` button. 

(Or using some other terms...?)